cedar-policy/benches/cedar_benchmarks.rs

```diff
@@ -23,6 +23,8 @@ use cedar_policy::{
 
 use criterion::{black_box, criterion_group, criterion_main, Criterion};
 
+// PANIC SAFETY unit tests
+#[allow(clippy::unwrap_used)]
```

Comment:
Do you know why this annotation wasn't required before?

Reply:
`cargo clippy` from the cmdline runs without the `#[cfg(test)]` enabled. In VSCodee it runs with `#[cfg(test)]` enabled. Raises the point of making sure we test this in CI with all features, double checking on that now